syntax = "proto3";

package product;

message Product {
    int32 id = 1;
    int32 category_id = 2;
    int32 brand_id = 3;
    int32 user_id = 4;
    string title = 5;
    string description = 6;
    float price = 7;
    string image_url = 8;
    int32 quantity = 9;
    float rating = 10;
    string created_at = 11;
    string updated_at = 12;
}
